### PR TITLE
update SMC v4

### PR DESCRIPTION
**Pull request by aloctavodia**

### CODE REVIEW

File: examples/samplers/SMC2_gaussians.ipynb [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "# Sequential Monte Carlo\n",
```
Comment (ricardoV94) [inline examples/samplers/SMC2_gaussians.ipynb:7]: <p>testval should be replaced by initval as the warning suggests </p><p> </p><p>Seems like in sample SMC we should suppress the Potential warning wheb sampling from the prior to start the population...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/334/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (aloctavodia) [reply examples/samplers/SMC2_gaussians.ipynb:7]: <p>testval replaced.</p><p> </p><p>I tried to filter the warning, but then the progressbar does not progress :-(</p><p> </p>

Comment (ricardoV94) [reply examples/samplers/SMC2_gaussians.ipynb:7]: > I tried to filter the warning, but then the progressbar does not progress :-

I mean internally in PyMC

Comment (aloctavodia) [reply examples/samplers/SMC2_gaussians.ipynb:7]: I tried to catch the warning inside the initialize_population function.

Comment (ricardoV94) [reply examples/samplers/SMC2_gaussians.ipynb:7]: How did you do it?

Comment (aloctavodia) [reply examples/samplers/SMC2_gaussians.ipynb:7]: see this https://github.com/pymc-devs/pymc/pull/5827

**Review by twiecki — APPROVED** [review]

**Pull request merged**